rustzx-z80/src/opcode/group_nonprefixed.rs

```diff
@@ -367,6 +371,7 @@ pub fn execute_normal(cpu: &mut Z80, bus: &mut impl Z80Bus, opcode: Opcode, pref
                     flags |= carry as u8 * FLAG_CARRY;
                     flags |= F3F5_TABLE[data as usize];
                     cpu.regs.set_flags(flags);
+                    cpu.regs.set_q(flags);
```

Comment:
TODO: Move to `set_flags` and make separate method for settings flags without q if needed